src/release.rs

```diff
@@ -68,9 +76,12 @@ impl BuildConfig {
                 cat_file(&self.install_directory.join("version.yml"))?;
                 Ok(channel.clone())
             }
-            DeployMethod::Local(_) => {
-                let image_tag = self.build()?;
-                Ok(image_tag)
+            DeployMethod::Local(_) => Ok(self.build()?),
+            DeployMethod::Commit(commit, _, _) => {
+                if self.build_type == BuildType::Skip {
+                    return Ok(commit.to_string()[..8].to_string());
```

Comment:
nit: do you need to convert it a string twice?
```suggestion
                    return Ok(commit[..8].to_string());
```